backend/src/test/java/com/crm/service/impl/LoginServiceImplTest.java

```diff
@@ -0,0 +1,126 @@
+package com.crm.service.impl;
+
+import com.crm.config.JwtTokenUtils;
+import com.crm.dto.response.JwtResponse;
+import com.crm.model.db.AccountEntity;
+import com.crm.model.db.RoleEntity;
+import com.crm.repository.AccountRepository;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContext;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Optional;
+
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+
+@SpringBootTest
+@RunWith(MockitoJUnitRunner.class)
+public class LoginServiceImplTest {
+    
+    @Mock
+    private AuthenticationManager authenticationManager;
+    
+    @Mock
+    private AccountRepository accountRepository;
+    
+    @Mock
+    private JwtTokenUtils jwtTokenUtils;
+    
+    @InjectMocks
+    private LoginServiceImpl loginService;
+
+    private static AccountEntity accountEntity;
+    private static Authentication auth;
+    private static SecurityContext secCont;
+    private static UserDetails userDetails;
+
+    private final Long ID = 1L;
+    private final String LOGIN = "Test48";
+    private final String EMAIL = "Test48";
+    private final String PASSWORD = "Test48";
+    private final boolean IS_ACTIVATED = true;
+    private final Timestamp LAST_FAILED_LOGIN = Timestamp.from(Instant.now());
+    private final Timestamp REGISTRATION_DATE = Timestamp.from(Instant.now());
+    private final Timestamp LAST_SUCCESSFUL_LOGIN = Timestamp.from(Instant.now());
+    private final int LOGIN_ATTEMPTS = 1;
+    private final RoleEntity ROLE = RoleEntity.builder().id(1L).name("admin").build();
+    private final String token = "token";
+    private final String refreshToken = "token";
```

Comment:
In CAPITALS should only be constants (private static final), whereas other field names should be in small letters.

Reply:
done